README.md

```diff
@@ -15,12 +15,18 @@ Registrator is a GitHub app that automates creation of registration pull request
 
 First, install the app on your package(s) as mentioned above.  The procedure for registering a new package is the same as for releasing a new version.
 
+1. Set the [Project.toml](Project.toml) version field in your repository to your new desired `version`.
+2. Comment `@JuliaRegistrator register()` on the commit/branch you want to release (e.g. like [here](https://github.com/JuliaComputing/Registrator.jl/issues/61#issuecomment-483486641) or [here](https://github.com/chakravala/Grassmann.jl/commit/3c3a92610ebc8885619f561fe988b0d985852fce#commitcomment-33233149)).
+3. If something is incorrect, adjust, and redo step 2.
+4. Finally, either rely on [TagBot](https://github.com/apps/julia-tagbot) to tag and make a github release or alternatively tag the release manually.
+
 Registrator will look for the project file in the master branch by default, and will use the version set in the Project.toml file via, for example, `version = "0.1.0"`. To use a custom branch comment with:
 
 ```
 @JuliaRegistrator register(branch=name-of-your-branch)
 ```
 
+More detailed information about the usage of Registrator can be found in the [full readme](full_readme.md)
```

Comment:
How about just integrating the contents of `full_readme.md` into the main README? I agree that it seems weird to have two different README files.